addons/stock/models/stock_orderpoint.py

```diff
@@ -172,6 +173,13 @@ def _compute_unwanted_replenish(self):
                 after_replenish_qty = orderpoint.product_id.with_context(company_id=orderpoint.company_id.id, location=orderpoint.location_id.id).virtual_available + orderpoint.qty_to_order
                 orderpoint.unwanted_replenish = float_compare(after_replenish_qty, orderpoint.product_max_qty, precision_rounding=orderpoint.product_uom.rounding) > 0
 
+    @api.depends('location_id')
```

Comment:
why does it depends of location_id?

Reply:
During the process of creating new records, the compute method is not invoked automatically unless I include the location_id in the depends attribute. However, for existing records, this is not a problem. As an alternative, we could also utilize the warehouse_id instead.